internal/api_ui/server.go

```diff
@@ -356,6 +356,13 @@ func (s *Server) CreateCredential(ctx context.Context, request CreateCredentialR
 		claimRequestProofs.Iden3SparseMerkleTreeProof = true
 	}
 
+	var credentialStatusType verifiable.CredentialStatusType
+	if request.Body.CredentialStatusType == nil || *request.Body.CredentialStatusType == "" {
+		credentialStatusType = verifiable.Iden3commRevocationStatusV1
```

Comment:
Same comment as in api/server.go.  Any value for request.Body.CredentialStatusType is acceptable or we have a set of allowed values?

Reply:
check for allowed values added